heartbeat/kamailio.in

```diff
@@ -477,20 +477,20 @@ kamailio_status() {
 
     errorfile=`mktemp`
     case ${OCF_RESKEY_proto} in
-    udp)  output=`$OCF_RESKEY_sipsak -s sip:monitor@$OCF_RESKEY_monitoring_ip:${OCF_RESKEY_port} -H localhost --transport udp>/dev/null 2>>$errorfile`
```

Comment:
You should leave these as-is. 1 (stdout) is meant to go to /dev/null, and stderr to $errorfile.